scripts/lint_jjb.py

```diff
@@ -318,6 +318,14 @@ def check_timed_trigger(data, in_file):
     rc = 0
     for _dir in args.dirs.split(','):
         for root, dirs, files in os.walk(_dir):
+            skip_dir = False
+            excludes = map(lambda s: s.strip(), args.exclude_files.split(','))
+            for exclude in excludes:
+                if "{s}{ex}".format(s=os.sep, ex=exclude) in root:
```

Comment:
This could potentially catch things that have similar names to excluded directories.